lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -2386,6 +2391,41 @@ export class StandardController implements IController {
         );
         return res;
     }
+
+    private logMultipleInstances(
+        performanceEvent: InProgressPerformanceEvent
+    ): void {
+        const clientId = this.config.auth.clientId;
+
+        // @ts-ignore
+        window.msal = window.msal || {};
+        // @ts-ignore
+        window.msal.clientIds = window.msal.clientIds || [];
+
+        // @ts-ignore
+        const clientIds: string[] = window.msal.clientIds;
+
+        for (const currentId of clientIds) {
+            if (currentId === clientId) {
+                this.logger.warning(
+                    "There is already an instance of MSAL.js in the window with the same client id."
+                );
+                // @ts-ignore
+                window.msal.clientIds.push(clientId);
+                collectInstanceStats(clientId, performanceEvent);
+                return;
+            }
+        }
+
+        if (clientIds.length > 0) {
+            this.logger.warning(
```

Comment:
Is this really something we want to warn on? There are production apps that use this model and it is supported.

Reply:
I thought they wanted to know when they are running multiple instances. Isn't that the ask?

Reply:
Multiple instances of the same clientId = not supported. Multiple instances for different clientIds = supported. We can track both metrics but we shouldn't log warnings on supported scenarios.

Reply:
I still thing a verbose log indicated the same would be appropriate.

Reply:
Set to verbose